### DISCUSSION TITLE
How can I reactivate my old github page after changing the username?

### DISCUSSION THREAD

**Discussion by geon0325** [community/community#28463] [Other Feature Feedback, Questions, & Ideas] [answered]
I have recently changed my username from **OLD** to **NEW**. Then, I changed the old github page repository name **OLD**.github.io to **NEW**.github.io. Now **NEW**.github.io works fine, but the previous **OLD**.github.io does not redirects to the new one (**NEW**.github.io). Interestingly, github.com/**OLD**/**OLD**.github.io redirects to github.com/**NEW**/**NEW**.github.io. How can I reactivate my old github page?

**OLD** and **NEW** are virtual usernames, just for example.

**Comment by jfullstackdev** [comment]
it's highly discouraged to change your username as it is your ID, it is part of the URL, 

it's quite tricky actually, you may realize that you have retired repos, 

in your case, you can't do that because if that will be taken by another user, then it should not redirect to the current one, I manually changed the root NEW.github.io,

there is also another rule that if that will be taken by another user, any repo that will have the same name will not redirect because, of course, it's already owned by the other user

**Reply by geon0325** [reply]
Thank you for your answers! I actually created a new account with username **OLD**. However, even if I try to make a new repository named **OLD**.github.io, it is refused due to `The repository OLD.github.io has been retired and cannot be reused.` Is there any way I can take the repository back in **OLD**?

**Reply by airtower-luna** [reply]
Possibly, it's part of a [system to prevent malicious takeovers](https://github.blog/2018-04-18-new-tools-for-open-source-maintainers/#popular-repository-namespace-retirement) after people changing or deleting their accounts. I have seen mentions of cases where support released the block, but I don't know under which circumstances they do or don't do that. If you want to try you need to [create a ticket](https://support.github.com/request).

**Reply by jfullstackdev** [reply]
> `The repository OLD.github.io has been retired and cannot be reused.` Is there any way I can take the repository back in **OLD**?

that is what I was mentioning, yup that's correct, file a ticket and / or hopefully a Staff see this too here at the Discussion

**Comment by airtower-luna** [comment, marked as answer]
You can't, plain and simple. The redirects on `github.com` are a courtesy to anyone who still has the old link, but they'll break as soon as someone takes your old username and creates a repository of the same name.

**Comment by geon0325** [comment]
Thank you @jdevstatic and @airtower-luna for the discussions! I have created a ticket and hope the issue is addressed.
